dataprofiler/profilers/column_profile_compilers.py

```diff
@@ -121,35 +121,43 @@ def update_profile(self, df_series):
         df_series = df_series.apply(str)
                 
         # If single process, loop and return
-        if not self.multiprocess_flag:
+        if pool is not None:
```

Comment:
Shouldn't this be `if pool is None` since this is for single process, and `pool` being not None indicted multiprocessing is occuring? Or am I misunderstanding